ravenframework/OutStreams/PrintInterfaces/FilePrint.py

```diff
@@ -148,6 +148,8 @@ def run(self):
       if self.options['type'] == 'csv':
         filename = dictOptions['filenameroot']
         rlzIndex = self.indexPrinted.get(filename,0)
+        if rlzIndex and rlzIndex >= len(self.sourceData[index]):
```

Comment:
I am not sure if these two lines are needed

Reply:
this is needed when the printing of the dataset is not finished (e.g. point set) and this is triggered in the multi run step (right before exiting the step). We check here the index and the length in case the printing is finished before reaching the ned of the step